packages/block-library/src/audio/block.json

```diff
@@ -1,4 +1,5 @@
 {
+	"apiVersion": 1,
```

Comment:
Do you think we should start with 2 instead? assuming the existing one is 1 or everything starts with `0` in tech? :P

Reply:
This is JavaScript! :D I don't have a strong opinion.

Reply:
What if we made the lightBlockWrapper API be `0`, and made the existing one use `null`? :trollface:

Reply:
On a more serious note, I think I prefer the lightBlockWrapper API being version 2, since that way we can say that `apiVersion: 2` is the second API version. It makes sense.

Reply:
Ok, I'll adjust it